### DISCUSSION TITLE
Welcome to scratchpad Discussions!

### DISCUSSION THREAD

**Discussion by GorkamWorka** [worldanvil/scratchpad#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lyraine-Alei** [comment]
Hello! I'm Lyraine and I live part-time in the Help Channels, or in general, I just never.stop.talking.

Oh! And I'm part of the Codex Team!

Anyway, Hi!

**Comment by Xuroth** [comment]
Hello!

I'm Xuroth (Stephan) and I help build stuff for WA! I don't interact too much on the Discord, outside of Inner Sanctum and Development Discussion, but I can often help answer questions, especially in regards to the Features I've built.

**Comment by jonaswaeber** [comment]
Hi!

On World Anvil I am known as SoulLink!

I guess I should do this as well... I help Dimitris with bug triage and lots of helpful and less helpful advice :) And I am a maintainer of this repository, active in the help channels and a member of the Inner Sanctum.

**Comment by RiverFang13** [comment]
Hello!

I'm RiverFang and much like Lyraine, you can often find me in the help channels or elsewhere throughout the Discord. I'm a member of the Inner Sanctum and one of the Codex Team

**Comment by CBairdAsh** [comment]
Hi!

I'm Kummer Wolfe ( CB Ash ) I quietly pitch in and help Dimitris with security / penetration testing / sanity checking on new features as time/life allows me. I also help out on the Discord as well. I'm a member of the Inner Sanctum and generally try to help how I can.
